physics/GFS_DCNV_generic.F90

```diff
@@ -35,6 +36,7 @@ subroutine GFS_DCNV_generic_pre_run (im, levs, ldiag3d, do_cnvgwd, do_ca,
       real(kind=kind_phys), dimension(im,levs), intent(inout) :: save_u
       real(kind=kind_phys), dimension(im,levs), intent(inout) :: save_v
       real(kind=kind_phys), dimension(im,levs), intent(inout) :: save_t
+      real(kind=kind_phys), dimension(im,levs), intent(out), optional :: save_tcp
```

Comment:
The use of "optional" is a little different in CCPP. Need to remove it and change the metadata file as well.

Reply:
It turns out that you were only able to get away with this because of a bug that I introduced when switching from the old to the new metadata format. See https://github.com/NOAA-GSD/ccpp-framework/pull/1.

Reply:
Ok, good to know. I see no problem with making it non-optional